modin/core/execution/dask/implementations/pandas_on_dask/partitioning/__init__.py

```diff
@@ -12,3 +12,19 @@
 # governing permissions and limitations under the License.
 
 """Base Modin Dataframe classes related to its partitioning and optimized for pandas on Dask execution."""
+
+from .partition import PandasOnDaskDataframePartition
+from .partition_manager import PandasOnDaskDataframePartitionManager
+from .virtual_partition import (
+    PandasOnDaskDataframeVirtualPartition,
+    PandasOnDaskDataframeColumnPartition,
+    PandasOnDaskDataframeRowPartition,
+)
+
+__all__ = [
+    "PandasOnDaskDataframePartition",
+    "PandasOnDaskDataframePartitionManager",
+    "PandasOnDaskDataframeVirtualPartition",
+    "PandasOnDaskDataframeColumnPartition",
+    "PandasOnDaskDataframeRowPartition",
```

Comment:
I think these classes are implementation details of the virtual partition so we shouldn't expose them.

Reply:
Don't you think that `PandasOnDaskDataframeVirtualPartition` class is an implementation detail of `PandasOnDaskDataframeColumnPartition` and `PandasOnDaskDataframeRowPartition` classes? From the point of view of the fact that the partition manager manipulates precisely these classes.

Reply:
A virtual partition can span either all rows/cols in the dataframe or a part of rows/cols. Column/RowPartition classes specify what type of the part (rows/cols) the virtual parition spans. In depend on the axis the virtual partition can contain row partitions or col partitions. It seems to me we already discussed this in one of the PRs. Or am I missing something? We should probably discuss it again with more audience.

cc @modin-project/modin-core

Reply:
Base class is useful for inheritance check, like `isinstance(foo, BaseVirtualPartitionWhateverName)`

Reply:
Let's expose all these partition classes for now but we should probably reconsider the implementation of the classes in future.